src/main/io/ledstrip.c

```diff
@@ -80,25 +80,23 @@
 
 PG_REGISTER_WITH_RESET_FN(ledStripConfig_t, ledStripConfig, PG_LED_STRIP_CONFIG, 0);
 
+#define COLOR_UNDEFINED 255
```

Comment:
Isn't a better place for this next to, or even inside the ```colorId_e``` enum?

Reply:
No. It's not a real color and shouldn't be part of the enum (and would change the count inappropriately), and it's only used in ledstrip.c so there's no need for it to be in the header.